diffpy/pdfmorph/pdfmorphapp.py

```diff
@@ -412,7 +468,178 @@ def main():
             pairlist, labels, rmin=pmin, rmax=pmax, maglim=maglim, mag=mag, rw=rw, l_width=l_width
         )
 
-    return
+    return morph_results
+
+
+def multiple_morphs(parser, opts, pargs, stdout_flag=True):
+    # Custom error messages since usage is distinct when --multiple tag is applied
+    if len(pargs) < 2:
+        parser.custom_error("You must supply FILE and DIRECTORY. Go to --help for usage.")
+    elif len(pargs) > 2:
+        parser.custom_error("Too many arguments. Go to --help for usage.")
+
+    # Parse paths
+    morph_file = Path(pargs[0])
```

Comment:
this looks like a lot of io handling which would be better in some kind of io function somewhere rather than in the app I think.

Reply:
I have factored some io handling in an io module, but both functions take in a lot of inputs to make work.